www/src/components/ModalHeader.js

```diff
@@ -21,7 +21,7 @@ export function ModalHeader({ text, setOpen }) {
           justify="center"
           width="2rem"
           height="2rem"
-          hoverIndicator="background-light"
+          hoverIndicator="fill-one"
```

Comment:
the name fill-one seems a little unclear to me, is there a way to make it a little more apparent what that color is acting as?

Reply:
Ask Nick, that's his responsibility

Reply:
We can bring it up with nick too, but for codebase cleanliness, I do think we should have a mapping of semantic names that correspond to his more systematic ones.